tv_apps/smart-home/js/home.js

```diff
@@ -130,9 +157,28 @@
       });
     },
 
+    _onPinInit: function(newCardButtonElem) {
+      newCardButtonElem.classList.add('new-card');
+      newCardButtonElem.classList.add('new-card-transition');
+      newCardButtonElem.addEventListener('transitionend', function onPinned() {
+        newCardButtonElem.classList.remove('new-card-transition');
+        newCardButtonElem.removeEventListener('transitionend', onPinned);
+      });
+      this.cardListElem.classList.add('card-slide');
+    },
+
+
     onCardInserted: function(card, idx) {
-      this.cardScrollable.insertNodeBefore(this._createCardNode(card), idx);
-      this.cardScrollable.focus(idx);
+      var newCardElem = this._createCardNode(card);
+      var newCardButtonElem = newCardElem.children[0];
+      this.cardScrollable.spatialNavigator.getFocusedElement().blur();
```

Comment:
Avoid manipulating `Scrollable.spatialNavigator`.
You can use `this.cardScrollable.currentItem.blur()`.

I found you just didn't read my comment before. Please read my comment again.